Formula/kcov.rb

```diff
@@ -7,6 +7,13 @@ class Kcov < Formula
   revision 1
   head "https://github.com/SimonKagstrom/kcov.git"
 
+  # add livecheck to avoid release tags like `pre-v40`
+  # which is tagged as latest release as well
```

Comment:
```suggestion
  # We check the Git tags because, as of writing, the "latest" release on GitHub
  # is a prerelease version (`pre-v40`), so we can't rely on it being correct.
```

This is just a small tweak to the explanatory comment, so I'll simply commit it and squash-merge this.